converter/group.py

```diff
@@ -18,7 +18,7 @@ def convert(fig_group):
 def post_process_frame(fig_group: dict, sketch_group: Group) -> Group:
     convert_frame_style(fig_group, sketch_group)
 
-    if fig_group["resizeToFit"]:
+    if "resizeToFit" in fig_group and fig_group["resizeToFit"]:
```

Comment:
Why this change at all?

In case it's needed, then maybe do:

```suggestion
    if fig_group.get("resizeToFit"):
```

Reply:
I got an error and assumed we don't have it for sections but I was wrong (it was just missing in my test). I'm undoing the change 👍